channels/rdpgfx/client/CMakeLists.txt

```diff
@@ -22,8 +22,8 @@ set(${MODULE_PREFIX}_SRCS
 	rdpgfx_main.h
 	rdpgfx_codec.c
 	rdpgfx_codec.h
-	rdpgfx_common.c
-	rdpgfx_common.h)
+	../rdpgfx_common.c
+	../rdpgfx_common.h)
```

Comment:
Since you are using rdpgfx_common in client and server, maybe it is a good idea to change CMake to create a static lib to avoid duplicate compilation.